services/server/src/models/serviceInstance/route.js

```diff
@@ -19,6 +19,10 @@ function setupRouter({ io, serviceInstanceId }) {
       res.status(StatusCodes.OK).json({ serviceInstance: serviceInstance.toJSON() });
     }));
 
+  router.get('/date', asyncHandler(async (req, res) => {
+    res.status(StatusCodes.OK).json({ date: Date.now() });
```

Comment:
Can we return this as an ISO timestamp? Even though it adds extra parsing (ISO is supported by both cleint and server), it makes it easier to debug whats going on.

Reply:
done